pkg/plugin/cmd/agent/upgrade/upgrade.go

```diff
@@ -0,0 +1,165 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package upgrade
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/DataDog/datadog-operator/pkg/apis/datadoghq/v1alpha1"
+	"github.com/DataDog/datadog-operator/pkg/plugin/common"
+
+	"github.com/spf13/cobra"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	image          string
+	upgradeExample = `
+  # upgrade the version of the datadog agent to latest
+  %[1]s upgrade
```

Comment:
this means that the image will be: `datadog/agent:latest` ?
this command can work only once? 
WDYT about providing an option `--latest` to set to latest? 
so for the command will be: `kubectl datadog agent upgrade --latest`

Reply:
sounds good